assets/src/ImageBlockExtension.js

```diff
@@ -95,7 +95,7 @@ const addExtraControls = function() {
               title={ __( 'Planet4 Image Options' ) }
               initialOpen={ true }
             >
-              <SelectControl
+              {/* <SelectControl
```

Comment:
Maybe just remove this block completely, instead of commenting it.

Reply:
I thought we didn't want to completely remove it, in case we need to put it back? 🤔 (hence the initial idea to hide it with CSS)

Reply:
yes, indeed. but it will be part of the git log, if we ever want to re-add it.

Reply:
Okay, sure! Removed 🙂